website/templates/emails/email_contributors_of_public_registrations.txt.mako

```diff
@@ -0,0 +1,12 @@
+Hello ${user.fullname},
+
+Sara and Brian N., please revise below.
```

Comment:
Yes. This hotfix solely prevents registrations that are public and created **after yet-to-be-chosen date** from being made private. 

As the epic comes to a close the need to set an official release date for Retractions and Embargoes is more important. If the need for a major refactor is identified I would (obviously) appreciate as much time before that release date to make fixes, re-write tests, and give QA ample time to play with this on staging. 

But, if we can pick a cutoff date for use by the hotfix and set an official release date for Retractions and Embargoes we can issue everything out in *one* email to all users of pending registration policy and behavior changes. I think this would be ideal but it will require a bit more granular planning than we have done so far.